highlight_go/main.go

```diff
@@ -63,9 +63,11 @@ func Print(src []byte, w io.Writer, p syntaxhighlight.Printer) error {
 		// TODO: Clean this up.
 		whitespace := string(src[lastOffset:offset])
```

Comment:
Ok, I will. Can you please give me push rights to [your fork](https://github.com/rocky/go), so that I can push commits into this PR. I'll ask you to review them before merging this.